### ISSUE TITLE
New version from Munglunch

### ISSUE THREAD

**Issue by finalsliver** [joev/SVUI-Temp#90]
Putting this here so no-one else has to ask.

Have you contacted Munglunch since his return (12 hourse since last update from him on Curse)

If so plans for the future? Collaboration or Secession of the project?

Link to the Curse Filelist for SVui. [[Here]](https://wow.curseforge.com/addons/supervillain-ui/files/)

(Again, mostly making this so other people have a place to check. I also realise it was only 12 hours since Mung's latest update, Don't know what's different between them at this point myself :p )

**Comment by drveoj** [comment]
Yep. He's even joined the Discord server. We'll sort out integrating the codebases over the next few days. :)

**Comment by finalsliver** [comment]
Good to know. Will stick with the unofficial until I see a note on integration. 

I wish you happy negotiations an many squashed bugs!
